scripts/linters/pylint_extensions_test.py

```diff
@@ -3340,3 +3340,113 @@ def __init__(self, fake_arg):
 
         with checker_test_object.assertNoMessages():
             checker_test_object.checker.visit_classdef(metaclass_node)
+
+
+class DisallowHandlerWithoutSchemaTests(unittest.TestCase):
+
+    def test_non_schema_handlers_for_request_args_raise_error(self):
+        checker_test_object = testutils.CheckerTestCase()
+        checker_test_object.CHECKER_CLASS = (
+            pylint_extensions.DisallowHandlerWithoutSchema)
+        checker_test_object.setup_method()
+
+        schemaless_class_node = astroid.extract_node(
+            """
+            class FakeClass(base.BaseHandler): #@
+                URL_PATH_ARGS_SCHEMAS = {}
+            """)
+
+        with checker_test_object.assertAddsMessages(
+            testutils.Message(
+                msg_id='no-schema-for-handler-args',
+                node=schemaless_class_node,
+                args=(schemaless_class_node.name)
+            )
+        ):
+            checker_test_object.checker.visit_classdef(schemaless_class_node)
+
+    def test_non_schema_handlers_for_url_path_args_raise_error(self):
```

Comment:
Ditto as other comment. Please check the test name again and fix it.

Reply:
Done